acceptance/new-user/sign-up.test.js

```diff
@@ -1,105 +1,115 @@
-const puppeteer = require('puppeteer');
-const config = require('config');
+const puppeteer = require("puppeteer");
+const config = require("config");
 
-const URLS = require('../../constants/urls');
+const URLS = require("../../constants/urls");
 
 const { HOME_PAGE, SIGN_UP_PAGE, MY_HOST } = URLS;
 
 let browser, page;
 
+function delay(time) {
+    return new Promise(function (res) {
+        setTimeout(res, time);
+    });
+}
+
 beforeAll(async () => {
-  browser = await puppeteer.launch({
-    // headless: false,
-    // slowMo: 250
-  });
-
-  const context = await browser.createIncognitoBrowserContext();
-
-  page = await context.newPage();
-  // await page.emulate(puppeteer.devices['iPhone 8']);
-  // await page.setViewport({
-  //   width: 1440,
-  //   height: 900,
-  //   deviceScaleFactor: 1,
-  // });
-  return page;
+    browser = await puppeteer.launch({
+        headless: false,
+        slowMo: 0,
+    });
+
+    const context = await browser.createIncognitoBrowserContext();
+
+    page = await context.newPage();
+    // await page.emulate(puppeteer.devices['iPhone 8']);
+    // await page.setViewport({
+    //   width: 1440,
+    //   height: 900,
+    //   deviceScaleFactor: 1,
+    // });
+    return page;
 });
 
 afterAll(async () => {
-  await browser.close();
+    await browser.close();
 });
 
+describe("New user navigates", () => {
```

Comment:
May we have little more description here !
Or does it make sense this way ? 🤔

Reply:
Yeah you're right, I'll add it right away